README.md

```diff
@@ -70,6 +72,33 @@ rewardbench-gen --model={}
 For more information, see `scripts/run_generative.py`. 
 The extra requirement for local models is VLLM and the requesite API for API models (OpenAI, Anthropic, and Together are supported).
 
+### Logging
+
+The CLI comes with multiple advanced saving features for **model outputs** and **accuracy scores**. 
+These can be tied in metadata to reward models you own or uploaded as separate datasets to HuggingFace, such as for rejection sampling.
+For example, the following command does both:
+```
+rewardbench --model vwxyzjn/reward_modeling__EleutherAI_pythia-14m --batch_size 128 --tokenizer=EleutherAI/pythia-14m --push_results_to_hub --upload_model_metadata_to_hf --chat_template raw
```

Comment:
Maybe include a link to a dummy HF dataset just to demonstrate what it looks like?